generation/schema/local-flags.json

```diff
@@ -9,5 +9,14 @@
   "next": {
     "description": "Displays the next page of output.",
     "char": "n"
+  },
+  "sleep": {
+    "description": "Determines time waited between request for tail command. Defaults at 1 second.",
+    "char": "q",
```

Comment:
I chose q because it was not used in any other flag in the cli. Any character that is chosen for sleep and since will be eliminated as options for all other flags even if the flag is used in another command.